core/trino-main/src/main/java/io/trino/operator/OperatorStats.java

```diff
@@ -115,6 +121,7 @@ public OperatorStats(
             @JsonProperty("outputPositions") long outputPositions,
 
             @JsonProperty("dynamicFilterSplitsProcessed") long dynamicFilterSplitsProcessed,
+            @JsonProperty("customMetrics") Map<String, Metric> customMetrics,
```

Comment:
Could we use `Metrics` instead of `Map<String, Metric>` ?
Or if we just want to use maybe `Map<String, Metric>` directly, we can consider getting rid of Metrics wrapper and put the merge method in some utils class.

Reply:
Sounds good, changed to use `Metrics` instead of `Map`.